data/campaigns/Heir_To_The_Throne/scenarios/13_The_Dwarven_Doors.cfg

```diff
@@ -160,7 +160,78 @@
         #
         {VARIABLE_OP true_entrance_location rand "1..2"}
 
-        # TODO: manually add snow detritus
+        [lua]
+            code=<<
+
+local wml_actions = wesnoth.wml_actions
```

Comment:
That moves it closer to being general-use. It's **very** map dependent as it appears here and I don't want to make it too east for a UMC to copy it and then wonder why it doesn't work in their map.

What it really needs is a full, general use implementation. But that should be a subject branch separated from any other issues.